orchagent/watermarkorch.cpp

```diff
@@ -94,6 +87,44 @@ void WatermarkOrch::doTask(Consumer &consumer)
     }
 }
 
+void WatermarkOrch::handleWmConfigUpdate(const std::string &key, const std::vector<FieldValueTuple> &fvt)
+{
+    if (key == "TELEMETRY_INTERVAL")
+    {
+        for (std::pair<std::basic_string<char>, std::basic_string<char> > i: fvt)
+        {
+            if (i.first == "interval")
+            {
+                m_telemetryInterval = to_uint<uint32_t>(i.second.c_str());
+            }
+            else
+            {
+                SWSS_LOG_WARN("Unsupported key: %s", i.first.c_str());
+            }
+        }
+    }
+}
+
+void WatermarkOrch::handleFcConfigUpdate(const std::string &key, const std::vector<FieldValueTuple> &fvt)
+{
+    if (key == "QUEUE_WATERMARK" || key == "PG_WATERMARK")
+    {
+        for (std::pair<std::basic_string<char>, std::basic_string<char> > i: fvt)
+        {
+            if (i.first == "FLEX_COUNTER_STATUS" && i.second == "enable" && m_isTimerEnabled == false)
+            {
+                m_telemetryTimer->start();
+                m_isTimerEnabled = true;
+            }
+            else if (i.first == "FLEX_COUNTER_STATUS" && i.second == "disable" && m_isTimerEnabled == true)
```

Comment:
If "FLEX_COUNTER_STATUS" of the first processed key, say PG_WATERMARK, is "enable", while the second "QUEUE_WATERMARK" is "disable", the telemetryTimer will not start.

Reply:
@mykolaf can you fix the logic?